src/dev.config.ts

```diff
@@ -6,14 +6,13 @@ import * as CleanWebpackPlugin from 'clean-webpack-plugin';
 function webpackConfig(args: any): webpack.Configuration {
 	const config = baseConfigFactory(args);
 	const { plugins, output } = config;
-	const location = path.join('dev', args.element.name);
 	const outputPath = output!.path as string;
 
-	config.plugins = [...plugins!, new CleanWebpackPlugin([location], { root: outputPath, verbose: false })];
+	config.plugins = [...plugins!, new CleanWebpackPlugin([outputPath], { root: outputPath, verbose: false })];
```

Comment:
Sorry just noticed this, we only want to clean the target for the mode that is being built. For example in this case, `path.join(outputPath, 'dev')`

Reply:
@rorticus ^